src/test/kotlin/BoardItemTest.kt

```diff
@@ -4,18 +4,18 @@ import org.junit.jupiter.api.Test
 
 class BoardItemTest {
     @Test
-    fun `지뢰일때 지뢰타입이 맞는지 테스트`() {
+    fun `지뢰값을 true로 줬을때 지뢰 enum값이 넘어와야함`() {
```

Comment:
```suggestion
    fun `지뢰이면 MINE이 된다`() {
```
테스트 코드에는 가급적이면 로직을 노출하지 않는 것이 좋아요. (이하 관련 내용 동일)